content/influxdb/v2.0/reference/cli/influxd/upgrade.md

```diff
@@ -9,26 +9,50 @@ products: [oss]
 ---
 
 Use the `influxd upgrade` command to upgrade an instance of InfluxDB 1.x to InfluxDB 2.0.
-This command copies all data in [databases](/influxdb/v1.8/concepts/glossary/#database) and 
+This command copies all data in [databases](/influxdb/v1.8/concepts/glossary/#database) and
 [retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp) (used in 1.x)
 over to [buckets](/influxdb/v2.0/reference/glossary/#bucket) (used in 2.0).
 
 {{% warn %}}
 Be sure to back up all data before upgrading with `influx upgrade`.
 {{% /warn %}}
 
+This command performs the following actions:
+
+1. Reads 1.x configuration file and creates a 2.x config file with matching options.
```

Comment:
Creates the InfluxDB 2.x configuration file using 1.x configuration file options.